wrapper/src/test/java/integration/refactored/container/tests/ReadWriteSplittingPerformanceTest.java

```diff
@@ -204,7 +204,7 @@ private double toMillis(double nanos) {
   }
 
   private Connection connectToInstance(Properties props) throws SQLException {
-    final String url = ConnectionStringHelper.getProxyWrapperUrl();
+    final String url = ConnectionStringHelper.getWrapperClusterEndpointUrl();
```

Comment:
Why do we need the direct cluster endpoint url instead of the proxy urls?

Reply:
We don't need a proxy url here. Performance tests don't work with connectivity.